spec/unit/mutest/integration_spec.rb

```diff
@@ -11,14 +11,15 @@
   end
 
   describe '.setup' do
-    subject { described_class.setup(kernel, name) }
+    subject { described_class.setup(requirer, name) }
 
-    let(:name)   { 'null'               }
-    let(:kernel) { class_double(Kernel) }
+    let(:name) { 'null' }
+    let(:requirer) { Mutest::Requirer.new }
 
     before do
-      expect(kernel).to receive(:require)
+      expect(requirer).to receive(:require)
         .with('mutest/integration/null')
+        .and_call_original
```

Comment:
Well I'm glad we fixed the issue. It is also nice though to not invoke a require within a test. If the boot-time require of `mutest/integration/null` ever did change then we would have an order dependent global state change due to `and_call_original`.